### PR TITLE
Update java-client with new APIs.

### PR DESCRIPTION
**Pull request by danielin917**
- Support for paginated query lambdas added
- New model support added for new integrations

### CODE REVIEW

File: src/main/java/com/rockset/jdbc/RocksetConnection.java [hunk 1]
```diff
@@ -544,6 +544,7 @@ QueryResponse startQuery(
   QueryPaginationResponse getQueryPaginationResults(String queryId, String cursor, int fetchSize)
       throws Exception {
 
+    final QueryRequest request = new QueryRequest();
```
Comment (sudhindra-rockset) [inline src/main/java/com/rockset/jdbc/RocksetConnection.java:547]: wondering where is this used?

Comment (danielin917) [reply src/main/java/com/rockset/jdbc/RocksetConnection.java:547]: It's not will remove

File: src/test/java/com/rockset/client/TestQueryLambda.java [hunk 2]
```diff
@@ -80,4 +84,56 @@ public void testSavedQueryCrud() throws Exception {
...
+    QueryLambdaTagResponse respQLTag = client.queryLambdas.create_0("commons", paginatedQueryName, reqQLTag);
+    Assert.assertEquals(respQLTag.getData().getTagName(), queryLambdaTag);
+
+    // Execute the QL and get only the first page.
+    ExecuteQueryLambdaRequest exReq = new ExecuteQueryLambdaRequest();
+    exReq.setPaginate(true);
+    exReq.setDefaultRowLimit(1);
+    exReq.setInitialPaginateResponseDocCount(1);
+    QueryResponse qr = client.queryLambdas.execute("commons", paginatedQueryName, queryLambdaTag, exReq);
+    System.out.println(qr.getResults().toString());
```
Comment (sudhindra-rockset) [inline src/test/java/com/rockset/client/TestQueryLambda.java:115]: may want to remove

**Review by sudhindra-rockset — APPROVED** [review]
couple of minor comments, test looks great! might be useful to get a lgtm from @scottwmorris too

**Review by scottwmorris — APPROVED** [review]
test looks good to me too

**Pull request merged**